pybamm/models/submodels/interface/open_circuit_potential/wycisk_ocp.py

```diff
@@ -5,7 +5,7 @@
 from . import BaseOpenCircuitPotential
 
 
-class DCHSOpenCircuitPotential(BaseOpenCircuitPotential):
+class WyciskOpenCircuitPotential(BaseOpenCircuitPotential):
```

Comment:
This class needs a docstring explaining what the model is + giving the reference (like you do in the notebook) and it needs to be included in the appropriate `.rst` file in the `docs/source/api` folder

Reply:
Done.